src/modules/organisation/api/organisation-query.param.ts

```diff
@@ -0,0 +1,35 @@
+import { AutoMap } from '@automapper/classes';
+import { PagedQueryParams } from '../../../shared/paging/index.js';
+import { IsEnum, IsOptional, IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { OrganisationsTyp } from '../domain/organisation.enum.js';
+export class OrganizationQueryParams extends PagedQueryParams {
```

Comment:
New line and a better name would be `FindOrganisationQueryParams`.

Reply:
✅